flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java

```diff
@@ -190,6 +176,47 @@ public void write(int b) throws IOException {
 		}
 	}
 
+	@Test
+	public void testSetSessionPropertyWithException() throws Exception {
+		Executor executor = mock(Executor.class);
+		doThrow(new SqlExecutionException("Property 'parallelism' must be a integer value but was: 10a"))
```

Comment:
We should avoid using Mockito according to the Flink coding guidelines: https://flink.apache.org/contributing/code-style-and-quality-common.html

```
Mockito-based tests tend to be costly to maintain in the long run by encouraging duplication of functionality and testing for implementation rather than effect
More details: https://docs.google.com/presentation/d/1fZlTjOJscwmzYadPGl23aui6zopl94Mn5smG-rB0qT8
```

Reply:
Thanks for reminding, I will update this pr ASAP.